github-for-jira.sd.yml

```diff
@@ -12,6 +12,15 @@ compose:
     ports:
       - 8080:8080
 
+computeClassification:
+  dataType:
```

Comment:
added all the data classification labels because that is a requirement for prod

Reply:
Thanks a lot tom! Surprisingly it doesn't seem to be a CREDO task. Is it a Service Linter requirement?